### PR TITLE
fix(breadcrumb): fixed "0 more items" aria-label

### PR DESCRIPTION
**Pull request by ValentinaKozlova**
## Previous Behavior
- on first load of the component "0 more items" is shown in aria-label of the overflow button
## New Behavior
- amount of items should be shown in aria-label of the overflow button

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30074/perf-test-react-components/Avatar-mount_base.html">639</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30074/perf-test-react-components/Avatar-mount.html">671</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30074/perf-test-react-components/Button-mount_base.html">298</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30074/perf-test-react-components/Button-mount.html">292</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30074/perf-test-react-components/Field-mount_base.html">1128</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30074/perf-test-react-components/Field-mount.html">1107</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30074/perf-test-react-components/FluentProvider-mount_base.html">714</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30074/perf-test-react-components/FluentProvider-mount.html">719</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30074/perf-test-react-components/FluentProviderWithTheme-mount_base.html">88</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30074/perf-test-react-components/FluentProviderWithTheme-mount.html">98</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30074/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">76</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30074/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">82</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30074/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">90</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30074/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">85</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30074/perf-test-react-components/MakeStyles-mount_base.html">873</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30074/perf-test-react-components/MakeStyles-mount.html">880</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30074/perf-test-react-components/Persona-mount_base.html">1799</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30074/perf-test-react-components/Persona-mount.html">1718</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30074/perf-test-react-components/SpinButton-mount_base.html">1467</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/30074/perf-test-react-components/SpinButton-mount.html">1418</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

File: packages/react-components/react-breadcrumb/stories/Breadcrumb/BreadcrumbWithTooltip.stories.tsx [hunk 1]
```diff
@@ -133,10 +133,10 @@ const MenuWithTooltip = (props: PartitionBreadcrumbItems<Item>) => {
   if (!isOverflowing && overflowItems && overflowItems.length === 0) {
     return null;
   }
-  const overflowItemsLength = overflowItems?.length ?? 0;
+  const overflowItemsCount = overflowItems ? overflowItems.length + overflowCount : overflowCount;
```
Comment (Hotell) [inline packages/react-components/react-breadcrumb/stories/Breadcrumb/BreadcrumbWithTooltip.stories.tsx:136]: **suggestion:** looks like good candidate to encapsulate this logic to generic function as it's duplicated in previous story

**Review by Hotell — APPROVED** [review]

**Review by marcosmoura — APPROVED** [review]

**Comment by Hotell** [comment]
pipeline blocked by react-charting https://github.com/microsoft/fluentui/pull/30117

**Pull request merged**